Tasks/Xcode/task.json

```diff
@@ -282,24 +283,24 @@
-        "NoTestResultsFound": "No test result files matching %s were found, so publishing JUnit test results is being skipped.",
+        "ProvProfileDetailsNotFound": "Failed to find details for the provisioning profile: %s",
+        "ProvProfileUUIDNotFound": "Failed to find the UUID for provisioning profile: %s",
+        "MultipleWorkspacesFound": "Multiple Xcode workspace matches were found. The first match will be used: %s",
+        "WorkspaceDoesNotExist": "The Xcode workspace was specified, but it does not exist or is not a directory: %s",
+        "UseXcprettyForTestPublishing": "When using xcodebuild, enable `Use xcpretty` to publish test results. For this build, no test results will be published.",
+        "NoTestResultsFound": "No test result files matching `%s` were found. Publishing JUnit test results will be skipped.",
         "XcodeSuccess": "Xcode task execution completed with no errors.",
-        "TempKeychainDeleteFailed": "Failed to delete temporary keychain %s created during the build.",
-        "ProvProfileDeleteFailed": "Failed to delete the provisioning profile %s.",
-        "ExportMethodNotIdentified": "Failed to identify the export method to use automatically from the archive file. This might cause errors during build or produce an invalid package. Set `Export Options` to `Specify` or `Plist`.",
-        "ExportOptionsPlistInvalidFilePath": "Export Options Plist file does not exist at %s. Provide the path to a valid Plist file.",
-        "SchemeRequiredForArchive": "Scheme needs to be specified to generate the package with xcodebuild archive and export.",
-        "WorkspaceOrProjectRequiredForArchive": "Workspace or Project path needs to be specified to generate the package with xcodebuild archive and export.",
-        "FailedToGenerateExportOptionsPlist": "Failed to automatically generate an Export Options Plist to export an IPA. Please specify an Export Options Plist in the Package Options.",
-        "LocateXcodeBasedOnVersion": "Locate the developer path for Xcode %s",
-        "FailedToLocateSpecifiedXcode": "Unable to resolve the developer path for Xcode %s. Set the %s environment variable on the agent machine, or set `Xcode version` to `Default` or `Custom`.",
-        "CantDetermineProvisioningStyle": "Provisioning style not found in a pbxProject file. Defaulting to automatic signing. If the project uses manual signing, auto export may fail. `Export Options` can be set to `Specify` or `Plist` as a workaround."
+        "TempKeychainDeleteFailed": "Failed to delete the temporary keychain created during the build: `%s`",
+        "ProvProfileDeleteFailed": "Failed to delete the provisioning profile `%s`.",
+        "ExportMethodNotIdentified": "Failed to automatically identify the export method to use from the archive file. This might cause errors during the build or produce an invalid package. Set `Export Options` to `Plist` or `Specify`.",
```

Comment:
I think `Export Options` is now `Export options`.

Reply:
Done... thanks!